app/src/main/res/drawable/vertical_ellipses.xml

```diff
@@ -12,12 +12,12 @@
             android:translateX="10.000000"
             android:translateY="33.000000">
             <path
-                android:fillColor="@android:color/black"
+                android:fillColor="#000000"
```

Comment:
AS was complaining that using color references here would cause issues on API < 24